README.md

```diff
@@ -44,7 +45,6 @@ Start by reading [how to use a template](docs/use_a_template.md), then check eac
 | [Postgres Monitor](postgresql/) | Monitor Postgres Server. CPU, Deadlocks, Data and more | [Ignacio Van Droogenbroeck](https://github.com/xe-nvdk) |
 | [Redis Monitor](redis/) | Monitor your Redis server. | [@russorat](https://github.com/russorat) |
 | [sFlow Traffic Monitor](sflow/) | Monitor your sFlow traffic. | [@russorat](https://github.com/russorat) |
-| [SNMP Monitoring](snmp/) | Dashboards showing metrics provided via SNMP protocol | [bonitoo.io](.) |
```

Comment:
This line was added to the main README probably after you checked out master, can you add it back in?

Reply:
@mhall119 - Just added this now in commit https://github.com/b3vis/community-templates/commit/aefe90f27e18a2e3ae316e96487fedc8185c6dcd